s3filepath/s3filepath.go

```diff
@@ -54,15 +55,17 @@ func (S3PathChecker) FileExists(path string) bool {
 // GetDataFilename returns the s3 filepath associated with an S3File
 // useful for redshift COPY commands, amongst other things
 func (f *S3File) GetDataFilename() string {
-	return fmt.Sprintf("s3://%s/%s_%s_%s.%s", f.Bucket.Name, f.Schema, f.Table, f.DataDate.Format(time.RFC3339), f.Suffix)
+	return fmt.Sprintf("s3://%s/%s/%s_%s_%s.%s", f.Bucket.Name, f.FilePath, f.Schema, f.Table, f.DataDate.Format(time.RFC3339), f.Suffix)
```

Comment:
It's not clear what this method does / why it's necessary. Why isn't `f.FilePath` the full path?

Reply:
Discussed offline and renamed.